src/main/scala/com/kinja/soy/SoyWrites.scala

```diff
@@ -75,6 +75,13 @@ trait DefaultSoyWrites {
   }
 
   /**
+   * Converter for Float types.
+   */
+  implicit object FloatSoy extends SoyWrites[Float] {
```

Comment:
We should change that to a `Double`. In my branch where I'm working on upgrading the Closure templates that didn't cause any problems.